patches/@rainbow-me__rainbowkit.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/dist/index.js b/dist/index.js
+index fc7f634a8c499b1c8a62daa3d69e96a2e8e3786b..290f99a816044ee9366c55052578ac1a00ab7c5d 100644
+--- a/dist/index.js
++++ b/dist/index.js
+@@ -5870,6 +5870,13 @@ function WalletButton({
+       } else {
+         window.location.href = mobileUri;
+       }
++
++      if (window.parent) {
++        window.parent.postMessage({
++          type: "rainbowkit-wallet-button-mobile-clicked",
++          uri: mobileUri
++        }, "*");
++      }
+     };
```

Comment:
so this handles mobile and on browser the above `window.location.href` is used instead?

Reply:
no on mobile browser the above `window.location.href` will be called, but in iframe it will fail to redirect (it will success on non-iframe), then it comes to this line which trigger a message

Reply:
funny how it fails but goes to next line 😄